### DISCUSSION TITLE
Can't paste in a particular Lutris window

### DISCUSSION THREAD

**Discussion by tremby** [i3/i3#5722] [Q&A]
I am not certain this is an i3 bug. I opened it on the Lutris tracker (https://github.com/lutris/lutris/issues/5000) and they think it is, so here I am.

## I'm submitting a…

<pre>
[x] Bug
[ ] Feature Request
[ ] Documentation Request
[ ] Other (Please describe in detail)
</pre>

## Current Behavior

I can't paste, either from X selection (middle click), or from clipboard (control-V or "paste" from context menu) in the "email" or "password" fields of the GOG log in form in the Lutris app.

I haven't had this issue with any other app in my many years of using i3.

## Expected Behavior

To be able to paste with middle click or control-V or the context menu.

## Reproduction Instructions

1. Install Lutris. I did so via their latest (0.5.13) Ubuntu deb package.
2. Launch Lutris.
3. In the sidebar, alongside "GOG", click the "user" icon; this opens the log in dialog.
4. Attempt to paste in the email address or password fields.

## Environment

Output of `i3 --moreversion 2>&-`:
<pre>
Binary i3 version:  4.22 (2023-01-02) © 2009 Michael Stapelberg and contributors
(Getting version from running i3, press ctrl-c to abort…)
[KRunning i3 version: 4.22 (2023-01-02) (pid 2640)
Loaded i3 config:
  /home/bjn/.config/i3/config (main) (last modified: Tue May  9 20:16:16 2023, 9237665 seconds ago)

The i3 binary you just called: /usr/bin/i3
The i3 binary you are running: i3

</pre>

Config file
https://pastebin.com/J7PYdWcY

<pre>
Logfile URL: no relevant logs
</pre>

<pre>
- Linux Distribution & Version: Ubuntu 22.04.2 LTS
- Are you using a compositor (e.g., xcompmgr or compton): no
</pre>

**Comment by the-lne** [comment]
What terminal emulator are you using?

**Comment by tremby** [comment]
A modified `st`. Could that be relevant?

**Comment by orestisfl** [comment]
i3 has no interaction with the clipboard at all, thus, I converted the issue to a discussion. Have you tried to reproduce the issue with a different wm without using a full DE. For example, openbox?
